server/lib/models/account.go

```diff
@@ -77,6 +71,19 @@ func (a *Account) StringId() string {
 	return strconv.FormatUint(uint64(a.Id), ACCOUNT_ID_BASE)
 }
 
+func (a *Account) Persona(dbif Dbif, channel proto.UUID) (*Persona, error) {
+	if p, ok := a.pmap[channel]; ok {
+		return p, nil
+	} else {
```

Comment:
__[gometalinter]__
* [golint] if block ends with a return statement, so drop this else and outdent its block (move short variable declaration to its own line if necessary)